### PR TITLE
Styling Changes to Nav Bar + Our Team page

### PR DESCRIPTION
**Pull request by adelwu-zz**
Screen Shot 2019-06-17 at 1 59 31 PM
Screen Shot 2019-06-17 at 1 59 40 PM
One weird bug I can't figure out is why the "Your Page" and "Logout" aren't showing up on the Our Team page despite having the addLoginOrLogoutLink() loader in the html. But I added some pictures and formatting to the Our Team page and did some nav bar clean up.

### CODE REVIEW

File: src/main/webapp/aboutus.html [hunk 1]
```diff
@@ -20,33 +20,55 @@
...
   </head>
-  </body>
-    <nav>
-      <ul id="navigation">
-        <li><a href="/">Home</a></li>
-        <li><a href="/aboutus.html">About Our Team</a></li>
-        <li><a href="/feed.html">Public Feed</a></li>
-      </ul>
-    </nav>
+  </body onload="addLoginOrLogoutLinkToNavigation();">
```
Comment (dhgarrette) [inline src/main/webapp/aboutus.html]: Looks like this should be `<body`, and that should fix the fact that some of the header links are different on this page.

**Review by dhgarrette — APPROVED** [review]
This is awesome!  Thanks, Adel!  I suggested one fix for an incorrect tag.  

I like that you're using `addLoginOrLogoutLinkToNavigation` ensure that the same processing is happening for the navigation links in every page, but since there's still a fair bit of repetition (e.g. each pages has the same "Home", "Our Team", and "feed" links), you might consider a next PR that adds a similar javascript function that builds the entire navigation and really try to avoid as much repeated code as possible.

**Comment by dhgarrette** [comment]
Also, make sure you do another "Squash and merge" since your branch has commits from your previous PR listed.  In general, I tend to make a new branch for each PR, and delete branches once the PR is merged, but that's not strictly necessary.

File: src/main/webapp/aboutus.html (hunk 1 above)
Comment (adelwu-zz) [reply src/main/webapp/aboutus.html]: ah yes, thank you!

**Pull request merged**